<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doublewidget</class>
 <widget class="QDialog" name="doublewidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="dishlist">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>581</height>
    </rect>
   </property>
   <widget class="QFrame" name="frame_deskinfo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>45</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="label_num">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>num</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="label_total">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>total</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="fexpandmain_table" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>45</y>
      <width>271</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:red;</string>
    </property>
   </widget>
  </widget>
  <widget class="tableview_item" name="tableView_m">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>410</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tableview_item</class>
   <extends>QTableView</extends>
   <header>tableview_item.h</header>
  </customwidget>
  <customwidget>
   <class>fexpandmain_table</class>
   <extends>QTableView</extends>
   <header>fexpandmain_table.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
